### ISSUE TITLE
Stroke/Shadow feature? - Unable to see white text on white

### ISSUE THREAD

**Issue by My-Name-Is-Adam** [White-Tiger/T-Clock#262]
I know you can't set a background colour but I would like to keep it at the default. 

Thank you.

**Comment by bearoxo** [comment]
Yea... I'm using T-Clock along with TaskbarX (app that makes the taskbar invisible)
It's hard to see the fonts without any font outline option

**Comment by blackcrack** [comment]
bullschit !

![Screen 00000](https://user-images.githubusercontent.com/7925717/127425343-edcea7d0-e6f7-4d24-bdeb-280eda2877e7.jpg)

@My-Name-Is-Adam you can confidently close it ..

by the way, did you see the last programming entry in the sourcefolder ?
this programm is not further supportet and it want works no more in Win11

**Comment by My-Name-Is-Adam** [comment]
> bullschit !
> 
> ![Screen 00000](https://user-images.githubusercontent.com/7925717/127425343-edcea7d0-e6f7-4d24-bdeb-280eda2877e7.jpg)
> 
> @My-Name-Is-Adam you can confidently close it ..
> 
> by the way, did you see the last programming entry in the sourcefolder ?
> this programm is not further supportet and it want works no more in Win11

I don't believe you actually read what I said because if you did you wouldn't have commented about the background colour option as that's not what I was asking about. 
Also, I am not using Windows 11 - I am using Windows 10 and don't plan on upgrading anytime soon. I believe this will also apply to many other people.
